CloudBoltAutomation/Functions/Start-CBOrder.ps1

```diff
@@ -1,22 +1,56 @@
+
 function Start-CBOrder 
 {
     <#
     .SYNOPSIS
-    
+    Starts an order in CloudBolt.
 
     .DESCRIPTION
-    
+    The `Start-CBOrder` function creates an order in CloudBolt and submits/starts it. You can use `Wait-CBOrder` to wait for the order to complete.
 
-    .EXAMPLE
-    
+    Pass a session object to the CloudBolt instance to use to the `Session` parameter (use `New-CBSession` to create a session object). Pass the ID of the group making the order to the `GroupID` parameter. Pass the blueprint ID to order to the `BlueprintID` parameter. If the build item being run by the blueprint has any parameters, pass them as a hashtable to the `BuildItemParameter`. Pass a name for the order to the `Name` parameter. Pass the name of the blueprint's resource (if any) to the `ResourceName` parameter. Pass any resource parameters as a hashtable to the `ResourceParameter` function.
 
-    
+    The request body generated by this function is visible in debug output. Set `$DebugPreference = 'Continue'` to see it.

```

Comment:
Would it be better to have the docs say to use the `-Debug` switch instead or say you can use one or the other? I feel using `$DebugPreference` is outside the scope of this function, as that's something you care about when scripting scripting multiple functions/cmdlets together. The scope of these docs is the function itself in which using `-Debug` is sufficient/cleaner.